libraries/browser-client/src/client/types.ts

```diff
@@ -285,4 +285,10 @@ export interface ContextOptions {
    * e.g. "Africa/Abidjan". Timezone should be wrapped to list, like ["Africa/Abidjan"].
    */
   timezone?: string[]
+
+  /**
+   * Inference time setting to use the non-streaming NLU variant. Set value to ["yes"] to enable,
+   * (or to ["no"] to disable for the current context if this parameter is enabled in the configuration).
+   */
+  nonStreamingNlu?: string[]
```

Comment:
Are we expecting external developers to use this interface? Isn't it a bit cumbersome and error-prone to control this feature with a rather low-level `["yes"]` instead of `true`?

Reply:
Now the nonStreamingNlu bit should be exposed as a boolean.